tasks.php

```diff
@@ -743,8 +743,8 @@ function handle_action_on_a_specified_task()
                     percent_complete = $edit_percent
```

Comment:
do we want to remove the $ variables and use sprintf exclusively to avoid %s being in a string throwing things off?

Reply:
Done.